threem.py

```diff
@@ -426,16 +426,11 @@ def __init__(self, _db, input_identifier_types=None,
             workset_size=25, metadata_replacement_policy=metadata_replacement_policy, **kwargs
         )
 
-    def process_batch(self, identifiers):
-        batch_results = []
-        for identifier in identifiers:
-            metadata = self.api.bibliographic_lookup(identifier)
-            result = self.set_metadata(identifier, metadata)
-            if not isinstance(result, CoverageFailure):
-                # Success!
-                result = self.set_presentation_ready(result)
```

Comment:
Don't need to explicitly call set_presentation_ready() anymore since that happens in handle_success().